### DISCUSSION TITLE
How does Carla implement and simulate weather changes?

### DISCUSSION THREAD

**Discussion by 853108389** [carla-simulator/carla#5478] [Q&A]
How does Carla implement and simulate weather changes? How to realize the change of point cloud data at different weather times? Where can I find reference materials? Thank you!

**Comment by KevinLADLee** [comment]
Hi, you could read this page for some details about the bp_weather.
https://carla.readthedocs.io/en/latest/tuto_M_custom_weather_landscape/#bp_weather
